.github/workflows/ci.yml

```diff
@@ -34,109 +35,80 @@ jobs:
       - name: checkout
         uses: actions/checkout@v3
       - name: toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.toolchain }}
           components: clippy, rustfmt
       - name: ubuntu dependencies
         run: |
           sudo apt-get update
           sudo bash scripts/install_ubuntu_dependencies.sh
-      - name: cargo fmt
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
+      - name: cargo format
+        run: cargo fmt --all -- --check
       - name: Install cargo-lints
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: cargo-lints
+        run: cargo install cargo-lints
       - name: Clippy check (with lints)
-        uses: actions-rs/cargo@v1
-        with:
-          command: lints
-          args: clippy --all-targets --all-features
+        run: cargo lints clippy --all-targets --all-features
   machete:
     name: machete
     runs-on: [ubuntu-20.04]
     steps:
       - name: checkout
         uses: actions/checkout@v3
       - name: toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.toolchain }}
           components: clippy, rustfmt
       - name: ubuntu dependencies
         run: |
           sudo apt-get update
           sudo bash scripts/install_ubuntu_dependencies.sh
-      - name: Install cargo-machete
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: cargo-machete
-      - name: cargo machete check
-        uses: actions-rs/cargo@v1
-        with:
-          command: machete
-  build:
-    name: cargo check
+      - name: cargo machete
+        run: |
+          cargo install cargo-machete
+          cargo machete
+  build-nightly:
+    name: cargo check for nightly
     runs-on: [self-hosted, ubuntu-high-cpu]
     steps:
       - name: checkout
         uses: actions/checkout@v3
       - name: toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ env.toolchain }}
-          profile: minimal
-          override: true
       - name: ubuntu dependencies
         run: |
           sudo apt-get update
           sudo bash scripts/install_ubuntu_dependencies.sh
+      - name: rustup show
+        run: rustup show
```

Comment:
Is this just for human-readable debugging output of some kind?

Reply:
Match a good idea just to confirm what rust deps are in place at build time.